src/QsCompiler/BondSchemas/BondSchemaTranslator.cs

```diff
@@ -126,7 +120,7 @@ qsBindingKind.Tag switch
                 Kind = qsCallable.Kind.ToBondSchema(),
                 FullName = qsCallable.FullName.ToBondSchema(),
                 Attributes = qsCallable.Attributes.Select(a => a.ToBondSchema()).ToList(),
-                Modifiers = qsCallable.Modifiers.ToBondSchema(),
+                Modifiers = new Modifiers { Access = qsCallable.Access.ToBondSchema() },
```

Comment:
Why doesn't this generate a warning; I thought Modifiers was obsolete? Does this mean we will need to update the schema version when we remove the deprecated pieces?

Reply:
This is `Microsoft.Quantum.QsCompiler.BondSchemas.Modifiers`, which is generated by the Bond schema. I don't think a new schema version is necessary, since we can keep the Bond versions of `Modifiers` and `AccessModifier` indefinitely, even after the other deprecated types are removed.